debezium-testing/debezium-testing-testcontainers/src/main/java/io/debezium/testing/testcontainers/ConnectorConfiguration.java

```diff
@@ -68,14 +66,10 @@ public static ConnectorConfiguration forJdbcContainer(JdbcDatabaseContainer<?> j
         return configuration;
     }
 
-    public static ConnectorConfiguration forMongoDbContainer(MongoDBContainer mongoDbContainer) {
-        final List<Integer> exposedPorts = mongoDbContainer.getExposedPorts();
-
+    public static ConnectorConfiguration forMongoDbContainer(MongoDbContainer mongoDbContainer) {
         ConnectorConfiguration configuration = new ConnectorConfiguration();
         configuration.with(CONNECTOR, "io.debezium.connector.mongodb.MongoDbConnector")
-                .with(HOSTS, "rs0/" + mongoDbContainer.getContainerInfo().getConfig().getHostName()
-                        + ":" + exposedPorts.get(0))
-                .with(AUTO_DISCOVER_MEMBERS, false);
+                .with(CONNECTION_STRING, mongoDbContainer.getNamedAddress().toString());
```

Comment:
this returns a `MongoDbContainer.Address` instance and the `toString()` call resolves to `"<host>:<port>"`, which is not a valid connection string.